docker/pg_tests/scripts/tests/full_backup_test.sh

```diff
@@ -35,3 +35,16 @@ psql -f tmp/scripts/amcheck.sql -v "ON_ERROR_STOP=1" postgres
 tmp/scripts/drop_pg.sh
 
 echo "Full backup success!!!!!!"
+
+# Also we test here WAL overwrite prevention as a part of regular backup functionality
+
+export WALG_PREVENT_WAL_OVERWRITE=true
+
+echo test > /tmp/test_file
+
+wal-g wal-push /tmp/test_file test_file
+wal-g wal-push /tmp/test_file test_file
+echo test1 > /tmp/test_file
+!wal-g wal-push /tmp/test_file test_file
```

Comment:
What is the reason for the exclamation mark? I think that it executes the last command that starts with wal-g. Does it check that rewriting was failed?

Reply:
This should negate WAL-G exit code.

Reply:
I checked it. It works on ubuntu if you have space between quotation mark and command.

like there
! wal-g wal-push something